e2e/quit-on-close.e2e.spec.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import util from 'util';
+
+import { test, expect } from '@playwright/test';
+import { ElectronApplication, _electron, Page } from 'playwright';
```

Comment:
Please import from `@playwright/test` instead of `playwright` (and possibly using `import type`).  Please see #3856 for details.